doc/lsst.meas.extensions.shapeHSM/hsm-moments.rst

```diff
@@ -0,0 +1,16 @@
+.. py:currentmodule:: lsst.meas.extensions.shapeHSM
+
+HSMMoments Plugins
+==================
+
+The purpose of the `HsmMoments` algorithms is to measure shapes of both the sources (or objects) and PSFs.
+The plugins wrap the implementation of the adaptive moments algorithm described in `Bernstein & Jarvis (2002) <http://adsabs.harvard.edu/abs/2002AJ....123..583B>`_ from `GalSim <https://github.com/GalSim-developers/GalSim>`_.
+This implementation is known to be more robust than :py:class:`SdssShape <lsst.meas.base.SdssShapeAlgorithm`> algorithm.
+
+The various plugins are specialized for sources and PSFs.
+For instance, `HsmSourceMomentsRoundPlugin` is specialized to use circular adaptive Gaussians to measure the moments of sources, while `HsmPsfMomentsRoundPlugin` is specialized to use circular adaptive Gaussians to measure the moments of sources.
```

Comment:
I don't see `HsmPsfMomentsRoundPlugin`, just the base class and the debiased one. And if it has `Psf` in the name, I don't think it's supposed to be `to measure the moments of sources`?

Reply:
That's a typo indeed. I'm dropping the second half of that sentence and rearranging the sentences around here.

Reply:
Did you drop that part?